broker-cli/commands/health-check.js

```diff
@@ -46,28 +48,46 @@ async function healthCheck (args, opts, logger) {
 
     const {
       engineStatus = [],
+      orderbookStatus = [],
       relayerStatus = STATUS_CODES.UNKNOWN
     } = await client.adminService.healthCheck({})
 
-    if (!engineStatus.length) {
-      logger.info(`No Engine Statuses Returned`.red)
-    }
+    const windowWidth = size.get().width
```

Comment:
we dont need to calculate these widths anymore. If we remove `colWidths` from healthcheckTable, it should base the widths off of the largest value (with some padding)